pkg/xmtp/message.go

```diff
@@ -35,6 +53,14 @@ func getContext(env *messageApi.Envelope) interfaces.MessageContext {
 				senderHmac = &parsed.SenderHmac
 			}
 		}
+	} else if messageType == topics.V3Conversation {
+		if message, err := parseGroupMessage(env.Message); err == nil {
+			*shouldPush = true
```

Comment:
We should probably get the `shouldPush` from the message eventually like we do in V2.